pages/content/migration-guidance.mdx

```diff
@@ -0,0 +1,504 @@
+- Can you identify a sequence to migrate the services if you have more than one?
+- You’ll need to explain to stakeholders that any data migration will incur some downtime. You can avoid this if absolutely necessary (see the Data migration section below).
+
+### Review your skill sets
+
+- Do you have the skills to migrate your services available in your department? You will need: 
+  - people with the skills to work with the infrastructure (defined in the DDAT framework as an [infrastructure devops engineer](https://www.gov.uk/guidance/development-operations-devops-engineer) role and also referred to as [Site Reliability Engineering](https://en.wikipedia.org/wiki/Site_reliability_engineering) in the wider technology marketplace), to enable the migration, change your pipelines§
+  - people with the skills to refactor your applications to accommodate the new hosting arrangement (i.e. software developers).
+- Do you need to outsource to a delivery partner?
+- Do you need to bring in contractors to supplement your team?
+- Do you need to procure [cloud migration expertise from the Digital Marketplace](https://www.digitalmarketplace.service.gov.uk/g-cloud/search?q=cloud%20migration) to assist with the migration?
+
+
+### Consider your department’s cloud hosting strategy
+
+- Understand your department's hosting standards and preferred cloud provider
+- Determine if there is a departmental strategy to follow
+- Determine if there is a departmental hosting platform available in-house. Some departments may have multiple hosting platforms for different types of workload. (for example one platform for general purpose web - hosting for OFFICIAL level services and one for data heavy workloads for data scientists)
+
+### Assess your security needs
+
+You will need to review the [threat model](https://www.ncsc.gov.uk/collection/building-a-security-operations-centre/onboarding-systems-and-log-sources/threat-modelling) of your service and re-assess your risks so that they can be accepted by your [senior responsible owner (SRO)](https://www.gov.uk/government/publications/the-role-of-the-senior-responsible-owner/the-role-of-the-senior-responsible-owner#key-project-leadership-roles)
+
+## Decide where to host your services
+
+There are many possible hosting options, but not all of them will necessarily make sense for your service. You should plan to spend time and effort exploring options and evaluating technical solutions according to your specific needs. 
+
+You should prefer known technologies with a proven track record like your public cloud provider’s virtual machines or basic container hosting offerings.
+
+You should consider services with higher levels of abstraction than Public Cloud IaaS, such as:
+
+- AWS Copilot (to provision ECS)
+- AWS ECS on Fargate
+- AWS Lambda
+- Google App Engine 
+- Google Cloud Run
+- Heroku 
+- Microsoft Azure App Service
+
+These are just some examples rather than endorsements.
```

Comment:
Add: Make sure things work for your use-case (or some such words). Feedback from BEIS was that people will run with what look like instructions or recommendations.